src/app/services/tree-node.service.ts

```diff
@@ -64,6 +64,11 @@ export class TreeNodeService {
       this._isLoading = true;
 
       this.apiEndpointService.getCall('projects').subscribe((projectsList) => {
+        if (!Array.isArray(projectsList)) {
+          alert(`Error while getting projects from your node (${this.config.getConfig('medco-node-url')}). It is correctly configured?`);
```

Comment:
@romainbou don't hesitate to change theses sentences if you think you can formulate better for the user.